### PR TITLE
Replicate hover effect from fabric website

### PR DESCRIPTION
**Pull request by mornir**
This UI library is great! Thanks a lot! I hope it will get more popular because it deserves it 💯

### CODE REVIEW

File: docs/components/DocsContent.vue [hunk 1]
```diff
@@ -54,10 +54,11 @@
     display: inline-block;
     text-decoration: none;
     margin-right: 28px;
+    cursor: pointer;
   }
 
   .header__nav--item:hover {
-    font-weight: 600;
+    transform: scale(1.1);
```
Comment (aidewoode) [inline docs/components/DocsContent.vue]: Hi,  I think ```font-weight: 600``` and ```transform: scale(1.1)``` have pretty much the same effect when hover a link. So there is no need to use ```transform scale``` to replace ```font-weight```. Or you have some others considerations ?

Comment (mornir) [reply docs/components/DocsContent.vue]: Yes, you're right. There's no need to use a transform here. On https://developer.microsoft.com/en-us/fabric#/components/button, 'font-weight: 600' is also applied on hover, but the effect looks more "centered". I wonder how they achieve that effect.

Comment (aidewoode) [reply docs/components/DocsContent.vue]: I know how to make the effect looks more "centered".

```css
.header__nav--item {
  display: inline-block;
  text-decoration: none;
  margin-right: 28px;
  cursor: pointer;
  text-align: center; /* need this one */
}
```

**Comment by aidewoode** [comment]
Thanks a lot 😃

**Pull request merged**